support.js

```diff
@@ -230,6 +230,7 @@ function testForKeySystem(ks, required) {
 testForClass(window, 'HTMLMediaElement', true);
 testForClass(window, 'MediaSource', true);
 testForClass(window, 'Promise', true);
+testForClass(window, 'Uint8Array', true);
```

Comment:
Would you also add Uint8Array to shaka.player.Player.isBrowserSupported()?  Ideally, that and this should agree.